sys/include/xtimer/implementation.h

```diff
@@ -211,22 +211,24 @@ static inline void xtimer_periodic_wakeup(xtimer_ticks32_t *last_wakeup, uint32_
     _xtimer_periodic_wakeup(&last_wakeup->ticks32, _xtimer_ticks_from_usec(period));
 }
 
-static inline void xtimer_set_wakeup(xtimer_t *timer, uint32_t offset, kernel_pid_t pid)
+static inline void xtimer_set_wakeup(ztimer_t *timer, uint32_t offset,
```

Comment:
:raised_eyebrow: ? Is that correct?

Reply:
nope, the script shouldn't hava run here but I thought I had fixed this...